standards/application/webrtc.md

```diff
@@ -0,0 +1,331 @@
+
+## Wire Protocol
+
+Signaling messages are encoded in a single Protocol Buffers message type. All messages share this format, distinguished by a message type field. The protobuf schema (proto3) is defined below:
+
+```proto
+syntax = "proto3";
+
+package waku.webrtc.v1;
+
+message WakuWebRTCMessage {
+  enum MessageType {
+    OFFER = 0;       // Initiator to responder: proposes a new WebRTC connection
+    ANSWER = 1;      // Responder to initiator: accepts and answers an OFFER
+    CANDIDATE = 2;   // Either peer to the other: conveys an ICE candidate (Trickle ICE, RFC 8838)
+    REJECT = 3;      // Responder to initiator: rejects an OFFER (connection refused)
+    BUSY = 4;        // Responder to initiator: rejects an OFFER because the responder is busy
+    END = 5;         // Either peer to the other: terminates an ongoing session (hang-up or cancel)
+    // (Future extensions could add ACK or other message types if needed)
+  }
+
+  MessageType message_type = 1;
+  bytes session_id = 2;               // Unique identifier for this session (generated by initiator)
+  optional bytes initiator_peer_id = 10;  // Waku peer ID of the initiator (caller)
+  optional bytes responder_peer_id = 11;  // Waku peer ID of the responder (callee)
+  optional bytes webrtc_data = 20;    // Encoded WebRTC data (SDP offer/answer or ICE candidate).
+}
+```
+
+### Message Fields
+
+- **message_type**: Indicates which signaling message this is (Offer, Answer, Candidate, etc.) and therefore what role and data to expect.
+- **session_id**: A random identifier for the session, generated by the initiator and included in all messages related to that session. It **MUST** be sufficiently unique (e.g. a UUID or 128-bit random value) such that concurrent sessions do not collide. Peers use this to tie incoming messages to the correct connection context.
+- **initiator_peer_id** *(optional)*: The libp2p/Waku peer ID of the initiating peer. This can be included in the Offer message, and subsequent messages **MAY** include it for clarity. It is mainly for informational or validation purposes, since the initiator is implicitly the sender of the initial Offer.
+- **responder_peer_id** *(optional)*: The peer ID of the responding peer (the target of the Offer). In an Offer, this identifies the intended recipient. In responses, it may be included as well. This is also primarily for information/routing confirmation, since the responder is typically the recipient of the initial Offer message.
+- **webrtc_data** *(optional)*: Binary payload carrying the WebRTC signaling data when applicable. The content depends on `message_type`:
+  - **For `OFFER`**: this field contains the WebRTC SDP Offer generated by the initiator (`RTCPeerConnection.createOffer()`), encoded as bytes. Typically this is the UTF-8 encoded SDP text (it **MAY** be compressed or binary encoded if both sides agree, but plain SDP text is simplest).
```

Comment:
```suggestion
- **webrtc_data** *(optional)*: Binary payload carrying the WebRTC signaling data when applicable. The content depends on `message_type`:
  - **For `OFFER`**: this field contains the WebRTC SDP Offer generated by the initiator (`RTCPeerConnection.createOffer()`), encoded as bytes straight from UTF-8..
```

be clear on the expected behaviour. If you want to open to compression, mention it as a potential improvement.